src/components/Dashboard/FilingHistoryList.vue

```diff
@@ -400,6 +400,7 @@ import { ActionBindingIF, ApiFilingIF, CorrectionFilingIF, DocumentIF, HistoryIt
   from '@/interfaces'
 import { AllowableActionsMixin, DateMixin, FilingMixin } from '@/mixins'
 import { EnumUtilities, LegalServices } from '@/services/'
+import {mapGetters} from "vuex";
```

Comment:
Linting should have caught this.-
- need a space after { and before }
- should be single quotes
- no semi-colon at end

Reply:
Good catch, I've got figure out how to tweak my editor so it automatically adds the import statements using the correct syntax.